admin-guides/sprint-calls-prep.md

```diff
@@ -0,0 +1,19 @@
+# Prep for Sprint Calls
+
+## Who does this Prep
+The moderator for the upcoming week's sprint calls
+
+## Steps to Take
+
+**Before the Call**
+
+1. Set up the sprint issue, using the [sprint issue template](https://github.com/ipfs/pm/blob/master/templates/sprint-issue.md), on Wednesday of the preceding week so people can start proposing agenda items
+1. Set up the hangouts links and post them on the sprint issue ticket
+1. Solicit IRC for agenda items
+
+**During the Calls**
+1. Act as moderator/facilitator on the all-hands call
```

Comment:
Needs a line feed here, otherwise the rendered version gets wonky

![image](https://cloud.githubusercontent.com/assets/1211152/18057724/16630c52-6e09-11e6-9dad-0e6a19feee1f.png)